packages/aws-cdk-lib/aws-appsync/README.md

```diff
@@ -232,6 +232,83 @@ httpDs.createResolver('MutationCallStepFunctionResolver', {
 });
 ```
 
+### EventBridge
+
+GraphQL schema file `schema.graphql`:
```

Comment:
Not being familiar with the event bridge integration, based on the mapping templates/schema, it looks like most of these types are just raw inputs/outputs from event bridge. Can we explain that in the docs a little bit?